### DISCUSSION TITLE
Unreadable current bookmark in "Darker" theme

### DISCUSSION THREAD

**Discussion by Freple** [sumatrapdfreader/sumatrapdf#4155] [Q&A] [answered]
Hello! As you can see from the image, when navigating through a document in the "Darker" theme, the bookmark you're currently on appears in black text and is thus very difficult to read.

![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/53348901/33f965a4-f98b-4ef4-8954-1ecbf79286a9)

Is there any way to fix this without switching to a lighter theme? I looked through the documentation about the settings.txt file but found nothing.

Thanks in advance!

**Comment by GitHubRulesOK** [comment, marked as answer]
No it is a byproduct of simple colour inversion. Fixing will require custom colour handling of the MS VS coded  wrapper.
Open issue is https://github.com/sumatrapdfreader/sumatrapdf/issues/3849

**Comment by plainee** [comment]
hope the dark mode bookmark text will be visible in the next version.

**Reply by GitHubRulesOK** [reply]
See my comment in the linked #3849 

![image](https://github.com/user-attachments/assets/811cc009-0e01-4a5a-b60f-240e95c1f2e9)
